packages/ringcentral-widgets/components/Dialog/index.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classnames from 'classnames';
+
+import styles from './styles.scss';
+import i18n from './i18n';
+import Button from '../Button';
+import CloseIcon from '../../assets/images/CloseIcon.svg';
+
+function FlatButton({
+  className, disabled, onClick, children
+}) {
+  return (
+    <div
```

Comment:
May I ask why not using <button /> tag here?

Reply:
I only separated the original Dialog implementation from the Modal logic without changing the Dialog component itself to ensure backward compatibility. I'd suggest creating a new Dialog component with further refactoring and redesign later.

Reply:
OK, I got the main idea.